pkgs/applications/science/biology/spades/default.nix

```diff
@@ -17,6 +17,11 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "${pname}-${version}/src";
 
+  CXXFLAGS = [
```

Comment:
```suggestion
  env.CXXFLAGS = toString [
```

I'd prefer that, but certainly not a blocker for merging.

Reply:
Sorry. Will update.